### PR TITLE
Dependency Upgrades

### PR DESCRIPTION
**Pull request by blimmer**
* [x] ember-cli (lots of good stuff in here)
* [x] ember-getowner-polyfill

### CODE REVIEW

File: addon/components/ember-islands.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import Ember from 'ember';
-import getOwner from 'ember-getowner-polyfill';
-const { $, assert, Component } = Ember;
+const { $, assert, Component, getOwner } = Ember;
```
Comment (blimmer) [inline addon/components/ember-islands.js]: It's a true polyfill now.

File: bower.json [hunk 2]
```diff
@@ -1,18 +1,4 @@
...
-    "ember-qunit-notifications": "0.1.0",
-    "ember-resolver": "~0.1.20",
-    "jquery": "~1.11.3",
-    "loader.js": "ember-cli/loader.js#3.2.1",
-    "qunit": "~1.20.0"
-  },
-  "resolutions": {
-    "ember": "~2.7.0"
-  }
+  "dependencies": {}
```
Comment (blimmer) [inline bower.json]: This is so nice. Very excited to see this.

**Comment by mitchlloyd** [comment]
Looks great. Thanks!

**Comment by mitchlloyd** [comment]
I'm being killed by Codeship... no branch build status showing up. Tested this locally and it works. I see that I'll need to cleanup some `_lookupFactory` deprecation spew soon.

**Comment by blimmer** [comment]
It might be because this is building from a fork - I've seen weirdness around that wrt travis.

**Pull request merged**